#40 - new design

brzozka opened the pull request:
Implemented new design into the LMT website.
New design [Figma](https://www.figma.com/design/1NPJfmUiOEyaZK1S1zYRes/LP-LMT%232?node-id=0-1&t=WB5uoqRILzd9oVku-1)

Main desktop view:
![prmltdesk](https://github.com/blumilksoftware/lmt/assets/39692087/2a53e87d-3437-4d35-b331-ee5b34e2feef)
Main mobile view:
![Screenshot from 2024-07-05 14-39-46](https://github.com/blumilksoftware/lmt/assets/39692087/08286570-7a67-4e73-bf5c-297714f3e064)
Previous meetup view:
![Screenshot from 2024-07-05 14-40-07](https://github.com/blumilksoftware/lmt/assets/39692087/8dbb16bd-5417-480a-a5df-9b7d675bc055)

Each meetup data is stored in separate folder located in `assets/meetups/`. It consist of `data.json` file and images folder with subfolders for clarity. 

This should close #40.

kamilpiech97 commented on public/pastMeetup.html:
```diff
@@ -0,0 +1,586 @@
+            <div class="flex-col">
+              <h2 class="text-5xl font-bold">Galeria</h2>
+              <p class="pt-4 text-base text-neutral-200">
+                Autor zdjęć: Piotr Kowalski, Jolanta Druga
```
```suggestion
                Autorzy zdjęć: Piotr Kowalski, Jolanta Druga
```

brzozka replied:
Fixed.

kamilpiech97 commented on public/pastMeetup.html:
```diff
@@ -0,0 +1,586 @@
+              <a href="#speakers" class="block px-3 py-2 hover:bg-gray-700 hover:text-white"
+                >Prelegenci</a
+              >
+              <a href="#agenda" class="block px-3 py-2 hover:bg-gray-700 hover:text-white"
+                >Agenda</a
+              >
```
Maybe gallery should be added too?

brzozka replied:
I forgot to add it after creating gallery section, fixed.

EwelinaSkrzypacz commented on .prettierignore:
```diff
@@ -0,0 +1,4 @@
+**/*.lock
+**/*.yml
+**/*.yaml
+**/*.md
```
```suggestion
**/*.md

```

brzozka replied:
Fixed.

EwelinaSkrzypacz commented on public/assets/meetups/2024-09-11-lmt-02/data.json:
```diff
@@ -0,0 +1,98 @@
+{
+  "title": "II legnicki meetup technologiczny",
+  "datetime": "2024-09-11 18:00:00",
```
According to lmt Facebook page, the meetup's date is 2024-09-12

brzozka replied:
The data I provided is just for testing. File with correct data should be added before the next meetup.

EwelinaSkrzypacz commented on public/index.html:
```diff
@@ -1,284 +1,455 @@
+              <h2 class="text-2xl font-bold">Dla kogo?</h2>
+              <p class="mt-5 max-h-[114px] text-base leading-5 tracking-tight">
+                Spotkania są dostępne dla każdego, bez względu na stopień doświadczenia zawodowego
+                czy wykorzystywaną technologię. Zapraszamy programistów grafików, administratorów,
```
```suggestion
                czy wykorzystywaną technologię. Zapraszamy programistów, grafików, administratorów,
```

EwelinaSkrzypacz commented on public/index.html:
```diff
@@ -350,154 +540,250 @@ <h2 class="text-4xl font-bold mb-8">Dla kogo?</h2>
+            <div class="flex flex-col px-4 text-left lg:w-full">
+              <div class="text-5xl font-bold lg:mt-8">Poprzednie edycje</div>
+              <div class="text-md w-3/4 pt-6 lg:mx-auto lg:w-full lg:px-0 lg:text-base">
+                Sprawdź czy nic Cię nie ominęło
```
```suggestion
                Sprawdź, czy nic Cię nie ominęło
```

EwelinaSkrzypacz commented on public/index.html:
```diff
@@ -350,154 +540,250 @@ <h2 class="text-4xl font-bold mb-8">Dla kogo?</h2>
+                  name="company"
+                  id="company"
+                  class="block w-full rounded-md border-2 bg-zinc-300 px-4 py-3 text-gray-900 shadow-sm placeholder:text-gray-400 sm:leading-6"
+                  placeholder="Nazwa Firmy"
```
```suggestion
                  placeholder="Nazwa firmy"
```

EwelinaSkrzypacz reviewed (commented):
1. Word "sekund" is not at the same height as other words like "dni", "godzin", "minut"

![image](https://github.com/blumilksoftware/lmt/assets/56546832/21910600-ed35-4fae-8012-e1483d893cd3)

2. On design we have here some gradient background, but on implementation not. Is this okay for us?

Implementation:

![image](https://github.com/blumilksoftware/lmt/assets/56546832/29571139-2b59-4adc-aafc-b2da8ce635a5)

Design:

![image](https://github.com/blumilksoftware/lmt/assets/56546832/4b64c787-b9c0-4de3-8ec1-bd7c2f0d7e8b)

3. On design the space between logo and label is bigger that on implementation

Implementation:

![image](https://github.com/blumilksoftware/lmt/assets/56546832/234a67ae-6b79-4fda-9bff-7f0293819fad)

Design:

![image](https://github.com/blumilksoftware/lmt/assets/56546832/19bc2587-12a9-4f23-a3d0-abbfb094329b)

4. I think that we should add labels on register page. Or maybe floating labels? That is the question to graphic designer, please ask Natalia about it.

5. Errors on register forms don't look like errors.

![image](https://github.com/blumilksoftware/lmt/assets/56546832/33825edf-108c-4b7c-a018-357e716bbb79)

6. On design we have "Patronat" label, but on implementation we have "Sponsor". 

Design:

![image](https://github.com/blumilksoftware/lmt/assets/56546832/3ddb2a84-fbe3-4684-a862-ad174d727f79)

Implementation:

![image](https://github.com/blumilksoftware/lmt/assets/56546832/63416497-554d-41c4-aba6-a1e9382f3749)

7. On design the register form looks better

Design:

![image](https://github.com/blumilksoftware/lmt/assets/56546832/cd563a2c-9ab6-4293-bdfb-a2a3290ce3b4)


Implementation:

![image](https://github.com/blumilksoftware/lmt/assets/56546832/e33005a1-efef-4d03-af2c-52c33347a244)

AleksandraKozubal commented on public/index.html:
```diff
@@ -1,284 +1,455 @@
+      <div
+        class="mx-9 mb-32 mt-[90px] grid max-w-screen-2xl grid-cols-1 gap-7 text-white md:mt-44 lg:mx-3 lg:grid-cols-3 xl:px-14 2xl:mx-auto 2xl:px-0"
+      >
+        <div class="rounded-3xl border border-[#ad92e0] p-8">
```
This border color is used multiple times in the file, I think it would be better to create custom color for this in `tailwind.config.js`

brzozka replied:
Replaced hash color to custom from tailwind config.

AleksandraKozubal commented on public/index.html:
```diff
@@ -1,284 +1,455 @@
+            class="mt-8 flex h-[35px] flex-col pl-3 md:mt-12 md:flex-row md:divide-x md:divide-slate-100/25"
+          >
+            <div class="flex flex-row items-center pl-2 md:pr-8 lg:pl-0">
+              <image src="./images/icons/calendar.svg" class="size-6 self-center lg:size-4"></image>
```
Add alt tag here and in a few other images

brzozka replied:
Added missing alt tags.

AleksandraKozubal reviewed (commented):
That's picky thing, but on phones like Iphone 12/13 mini this title is too big for the screen
![image](https://github.com/blumilksoftware/lmt/assets/104600942/e376fcef-43c7-4213-b2aa-187168c54bc2)

brzozka commented:
> That's a bit picky thing, but on phones like Iphone 12/13 mini this title is too big for the screen ![image](https://private-user-images.githubusercontent.com/104600942/346478293-e376fcef-43c7-4213-b2aa-187168c54bc2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MzMzNTgsIm5iZiI6MTcyMDQzMzA1OCwicGF0aCI6Ii8xMDQ2MDA5NDIvMzQ2NDc4MjkzLWUzNzZmY2VmLTQzYzctNDIxMy1iMmFhLTE4NzE2OGM1NGJjMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwOFQxMDA0MThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZjY4NjE1MmExMWZkZjE1YzAyNWE1YWQ3ZmQ5OGU3ODNmMjJhOWM5ZDMwYWFmYjI5MjJmNzFlZWJlMDA4N2IyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.woZFte7vEG8YQOkCd-7h_CDxx_Et3vWtS6JNMWGeVvE)

I adjusted the styling to look better on smaller mobile devices, please check if everything looks fine.



> 1. Word "sekund" is not at the same height as other words like "dni", "godzin", "minut"
> 
> ![image](https://private-user-images.githubusercontent.com/56546832/346445341-21910600-ed35-4fae-8012-e1483d893cd3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MzMzNTgsIm5iZiI6MTcyMDQzMzA1OCwicGF0aCI6Ii81NjU0NjgzMi8zNDY0NDUzNDEtMjE5MTA2MDAtZWQzNS00ZmFlLTgwMTItZTE0ODNkODkzY2QzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDEwMDQxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFkMDk2MmRhMDZkYTIzODIyZGI4ZDZkMjMxNzViOThjM2I0NzMyNWRjN2M2NGNhODgzNzg2MGYzMzU3M2YwMzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.22mxCHowphOCM1OpAlU-MxiF_0uQ7yQVI-vLvVZfJ6I)
> 
> 2. On design we have here some gradient background, but on implementation not. Is this okay for us?
> 
> Implementation:
> 
> ![image](https://private-user-images.githubusercontent.com/56546832/346445899-29571139-2b59-4adc-aafc-b2da8ce635a5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MzMzNTgsIm5iZiI6MTcyMDQzMzA1OCwicGF0aCI6Ii81NjU0NjgzMi8zNDY0NDU4OTktMjk1NzExMzktMmI1OS00YWRjLWFhZmMtYjJkYThjZTYzNWE1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDEwMDQxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk1NTg2NmQzMDNiMWQ0ZmQ5NGRjZjY5ODA0Njc5MGJiOWU1ZDE1MzZiNTU0ZGFhOTcxNTJiNzAwZTY2ZTg0YzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.rWcloFBcTniVBrdm8zo9xvBR85-TeQRPnOGetb5rtqw)
> 
> Design:
> 
> ![image](https://private-user-images.githubusercontent.com/56546832/346446092-4b64c787-b9c0-4de3-8ec1-bd7c2f0d7e8b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MzMzNTgsIm5iZiI6MTcyMDQzMzA1OCwicGF0aCI6Ii81NjU0NjgzMi8zNDY0NDYwOTItNGI2NGM3ODctYjljMC00ZGUzLThlYzEtYmQ3YzJmMGQ3ZThiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDEwMDQxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU1Y2JiOGY1MzBiYTNhNzk2MDNhNDA3NmJiZWZhMjM4OTRjYTczNzNkN2I4MDBmOTFkMmUwM2UyN2QyZDFiZmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.I2MDn3zx5L_6BUQVwhiNUk4e64RLQ4XnXs69SbKuci8)
> 
> 3. On design the space between logo and label is bigger that on implementation
> 
> Implementation:
> 
> ![image](https://private-user-images.githubusercontent.com/56546832/346446336-234a67ae-6b79-4fda-9bff-7f0293819fad.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MzMzNTgsIm5iZiI6MTcyMDQzMzA1OCwicGF0aCI6Ii81NjU0NjgzMi8zNDY0NDYzMzYtMjM0YTY3YWUtNmI3OS00ZmRhLTliZmYtN2YwMjkzODE5ZmFkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDEwMDQxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFiMGZmOGM4N2U2ZTc0Mzc1MDExN2RiODM0ZDhiMzhjM2I4NzE3MDU4MmVhNWRjZTRlYjUxMzRlNWVkZWE0ZGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.7UiFr7kVTjaHGYX4XghZ04jfP12Sz_QJH3HUXKkgoOs)
> 
> Design:
> 
> ![image](https://private-user-images.githubusercontent.com/56546832/346446391-19bc2587-12a9-4f23-a3d0-abbfb094329b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MzMzNTgsIm5iZiI6MTcyMDQzMzA1OCwicGF0aCI6Ii81NjU0NjgzMi8zNDY0NDYzOTEtMTliYzI1ODctMTJhOS00ZjIzLWEzZDAtYWJiZmIwOTQzMjliLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDEwMDQxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJhY2EyODVlZDRlZGM1NTVhMjU5YzlkOTg5NmEwZDAyNTllMDZjNjk2MjUzYTU1NDk5MzEwMWNkZDk2ZDA5MzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.NF1BK9tVFDu9Fa1kdiqfOzDo2D0-gE_7n0-A8oSmY4Y)
> 
> 4. I think that we should add labels on register page. Or maybe floating labels? That is the question to graphic designer, please ask Natalia about it.
> 5. Errors on register forms don't look like errors.
> 
> ![image](https://private-user-images.githubusercontent.com/56546832/346448666-33825edf-108c-4b7c-a018-357e716bbb79.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MzMzNTgsIm5iZiI6MTcyMDQzMzA1OCwicGF0aCI6Ii81NjU0NjgzMi8zNDY0NDg2NjYtMzM4MjVlZGYtMTA4Yy00YjdjLWEwMTgtMzU3ZTcxNmJiYjc5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDEwMDQxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZkM2Q2MWVhZWNmZDVmM2Y3N2IyMmViNDBiN2MzNGFjMjc1YjY4Yjg5M2NmZGZmNzljY2NjMDgzOTBhNDY3MzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.JWWlKBv-UujSEH2qrV1yzf2AT6sCUXnXyMs2ENmFYjA)
> 
> 6. On design we have "Patronat" label, but on implementation we have "Sponsor".
> 
> Design:
> 
> ![image](https://private-user-images.githubusercontent.com/56546832/346453295-3ddb2a84-fbe3-4684-a862-ad174d727f79.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MzMzNTgsIm5iZiI6MTcyMDQzMzA1OCwicGF0aCI6Ii81NjU0NjgzMi8zNDY0NTMyOTUtM2RkYjJhODQtZmJlMy00Njg0LWE4NjItYWQxNzRkNzI3Zjc5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDEwMDQxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJjZGFiZjIxZmZiOWM0ZTNiNzk1MzhiM2NiMWNlMTdiMDIxYWI5MzNmZDg1NmUxM2E5OGJmNDFiMGJiMDFiODEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.eGTp-ch8wF1QNuy561fdseVKaQsOTxt6UJfBtiLazWY)
> 
> Implementation:
> 
> ![image](https://private-user-images.githubusercontent.com/56546832/346453379-63416497-554d-41c4-aba6-a1e9382f3749.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MzMzNTgsIm5iZiI6MTcyMDQzMzA1OCwicGF0aCI6Ii81NjU0NjgzMi8zNDY0NTMzNzktNjM0MTY0OTctNTU0ZC00MWM0LWFiYTYtYTFlOTM4MmYzNzQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDEwMDQxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk0ZWVkZGE3ZmMyMzc1MGVjZDJhMjk3M2Q0ZDM5MDA5Zjk4ZTRiNjQ5NWVkZThkNGJkMWZmY2FmZDA1MjA2MDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.M1PE90FqP8SypbNYZ8V1U7hB6xHbwJltljdyt6--Lr8)
> 
> 7. On design the register form looks better
> 
> Design:
> 
> ![image](https://private-user-images.githubusercontent.com/56546832/346453671-cd563a2c-9ab6-4293-bdfb-a2a3290ce3b4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MzMzNTgsIm5iZiI6MTcyMDQzMzA1OCwicGF0aCI6Ii81NjU0NjgzMi8zNDY0NTM2NzEtY2Q1NjNhMmMtOWFiNi00MjkzLWJkZmItYTJhMzI5MGNlM2I0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDEwMDQxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThhMDVkNTVlY2ZmNzJjZTU2NGM5Y2JmZmI0YzRmYTIwYWNhZGQ1NDYxYzA0NzdiZDZlNjdlMzk3NTY0MDI2NWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.DY19X49QRdGQFHdAn4Cyg9C_RTPJ8RjdyHlb6x3RJAc)
> 
> Implementation:
> 
> ![image](https://private-user-images.githubusercontent.com/56546832/346453724-e33005a1-efef-4d03-af2c-52c33347a244.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MzMzNTgsIm5iZiI6MTcyMDQzMzA1OCwicGF0aCI6Ii81NjU0NjgzMi8zNDY0NTM3MjQtZTMzMDA1YTEtZWZlZi00ZDAzLWFmMmMtNTJjMzMzNDdhMjQ0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDEwMDQxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTExMmI4NDJlYmU4MWIwY2JlYTdmODBiYWRhZGYxYzY5MjkwYTE2ZjgwZDFlODAwZjgzNGEyMmFhZTBhMDdlZDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hgYzB0xhW19mLU4RcLKktopBQO5nI6LRQj4_r-erMao)

1.That is some weird styling problem, but I tried to align it as close as I can.
2.Applying the styles from figma regarding the gradient and luminosity creates a problem where icons in boxes turn gray.
3. I adjusted the paddings to fit design.
4 and 5. Already discussed with Natalia and adjusted.
6. It was changed later into design and missed by me, fixed.
7. I've adjusted some fonts and styling to look better on mobile and smaller devices.

EwelinaSkrzypacz reviewed (commented):
1. I believe is should be here "Galeria" ;)

![image](https://github.com/blumilksoftware/lmt/assets/56546832/a3da4b61-bd59-456e-9c0d-f135ad5c0899)

2. I have impression that alt text is not really helpful; I mean that users with screen reader would hear  for example  "Gallery image 421937162_1503917793804547_2229482453578043715_n.jpg" 

3. I'm not a big fan of  text "Ups! Coś poszło nie tak." (we can find it register form, when user for example doesn't fill some input "Ups! Coś poszło nie tak. Sprawdź błędy poniżej i spróbuj ponownie.")

Maybe text "Rejestracja nie powiodła się. Sprawdź błędy poniżej i spróbuj ponownie." will be better?

4. I think we should give a little more space between inputs

![image](https://github.com/blumilksoftware/lmt/assets/56546832/70e3bf55-7f11-49d3-ac1b-70f1a64222ec)

brzozka commented:
> 1. I believe is should be here "Galeria" ;)
> 
> ![image](https://private-user-images.githubusercontent.com/56546832/346527003-a3da4b61-bd59-456e-9c0d-f135ad5c0899.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MzU5NTEsIm5iZiI6MTcyMDQzNTY1MSwicGF0aCI6Ii81NjU0NjgzMi8zNDY1MjcwMDMtYTNkYTRiNjEtYmQ1OS00NTZlLTljMGQtZjEzNWFkNWMwODk5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDEwNDczMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIxZmNhOTc4ZTJkYjQ0YjE5YmZkOWZkMTdiNTA1MTM0MmQ4ZWQzYjAyYTU1OGIxM2I4ZmQ5NDZmNDdiZDhjNjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-JhiA3aZZ9qTQCmWYZAJPmlkgUNc7vFJTI0cFCCCBqg)
> 
> 2. I have impression that alt text is not really helpful; I mean that users with screen reader would hear  for example  "Gallery image 421937162_1503917793804547_2229482453578043715_n.jpg"
> 3. I'm not a big fan of  text "Ups! Coś poszło nie tak." (we can find it register form, when user for example doesn't fill some input "Ups! Coś poszło nie tak. Sprawdź błędy poniżej i spróbuj ponownie.")
> 
> Maybe text "Rejestracja nie powiodła się. Sprawdź błędy poniżej i spróbuj ponownie." will be better?
> 
> 4. I think we should give a little more space between inputs
> 
> ![image](https://private-user-images.githubusercontent.com/56546832/346529685-70e3bf55-7f11-49d3-ac1b-70f1a64222ec.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MzU5NTEsIm5iZiI6MTcyMDQzNTY1MSwicGF0aCI6Ii81NjU0NjgzMi8zNDY1Mjk2ODUtNzBlM2JmNTUtN2YxMS00OWQzLWFjMWItNzBmMWE2NDIyMmVjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDEwNDczMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg3ZjdhOTUwOGMyODViOTRmZWI0MWYwM2Y5ZjhhMjE1M2ZhMTk3YjI1ZGVkZDg2ZThhOGNjYmFmODQ5MjAzMGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2LvmYiQ-0EnZ1tOgV5bhHz4YqIcCFiyd1wYU_FflQ74)

1. Fixed 

2. I've created an Issue regarding accessibility features.

3. Changed the information to more formal as You suggested.

4. Adjusted the spacing between elements.

EwelinaSkrzypacz commented on public/gallery.php:
```diff
@@ -0,0 +1,32 @@
+}
+
+$galleryImages = getGalleryImages($meetupId);
+echo json_encode(['images' => $galleryImages]);
```
```suggestion
echo json_encode(['images' => $galleryImages]);

```

brzozka replied:
Adjusted.

AleksandraKozubal commented on public/pastMeetup.html:
```diff
@@ -138,7 +144,7 @@
           <div
             id="event"
-            class="mx-auto max-w-4xl px-4 text-center text-5xl font-bold capitalize sm:pl-0 lg:w-full lg:text-7xl"
+            class="mx-auto max-w-4xl px-4 text-center text-[34px] font-bold capitalize xs:text-[40px] sm:pl-0 lg:w-full lg:text-7xl"
```
```suggestion
class="mx-auto max-w-4xl px-4 text-center text-4xl font-bold capitalize xs:text-5xl sm:pl-0 lg:w-full lg:text-7xl"
```

brzozka replied:
The values You provided are slightly bigger than the ones I set, which causes some problems on smaller mobile devices.
![Screenshot from 2024-07-10 21-23-23](https://github.com/blumilksoftware/lmt/assets/39692087/5570ceac-8e7c-461a-bb97-64b8b1bf39c6)
![Screenshot from 2024-07-10 21-23-40](https://github.com/blumilksoftware/lmt/assets/39692087/b3503bb7-e320-4c22-b96f-0f8ad0c99c5d)

AleksandraKozubal replied:
I missed changes due to page refreshing, so it's ok right now

EwelinaSkrzypacz commented on public/index.html:
```diff
@@ -1,284 +1,475 @@
+              <img
+                src="./images/icons/calendar.svg"
+                class="size-6 self-center lg:size-4"
+                alt="Ikona kalendarzu"
```
```suggestion
                alt="Ikona kalendarza"
```

brzozka replied:
Fixed.

krzysztofrewak commented on public/pastMeetup.html:
Maybe `meetup` would be a good enough name?

brzozka replied:
Image sizes are similar to the graphical design. For widescreen displays, we're maintaining a height of around 750px to ensure visual consistency. Since the previous site used smaller image components, I think uploading higher resolution images will be the best decision. By name meetup You mean to replace the Lineup with it?

krzysztofrewak replied:
I thought about renaming filename to `meetup.html`.

brzozka replied:
Okay, I changed the main file name to meetup.html

krzysztofrewak commented on public/assets/meetups.json:
```diff
@@ -1,3 +1 @@
-[
-  "./assets/meetups/2024-04-11-lmt-01.json"
-]
+["./assets/meetups/2024-04-11-lmt-01/data.json", "./assets/meetups/2024-09-11-lmt-02/data.json"]
```
```suggestion
[
    "./assets/meetups/2024-04-11-lmt-01/data.json",
    "./assets/meetups/2024-09-11-lmt-02/data.json",
]
```

brzozka replied:
I think this was caused by prettier, and my set line width. I changed the `printWidth` value from 100 to 80 to fix this problem, this will extend the files by couple of lines but make them more readable.

krzysztofrewak commented on public/assets/meetups/2024-09-11-lmt-02/data.json:
```diff
@@ -0,0 +1,98 @@
+{
+  "title": "II legnicki meetup technologiczny",
```
I would go with `"Legnicki meetup technologiczny #2"`, but please talk with @katarzynanowakblumilk about it.

brzozka replied:
The data I provided was to showcase all funcionality of the website. The data.json should be revised to represent correct information about upcoming meetup.

krzysztofrewak reviewed (changes requested):
I think these photos are too big or have too small resolution for this size:

![image](https://github.com/blumilksoftware/lmt/assets/10898728/853cc42a-5e7a-4a1f-aac2-128db5235d87)

EwelinaSkrzypacz reviewed (commented):
How about adding in the menu "Poprzednie edycje"?

brzozka commented:
> How about adding in the menu "Poprzednie edycje"?

Added Poprzednie edycje into navigation bar.
